src/components/Typography/index.tsx

```diff
@@ -0,0 +1,346 @@
+import { SerializedStyles, Theme, css } from '@emotion/react'
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import React, { FocusEvent, MouseEvent, forwardRef, useCallback } from 'react'
+import recursivelyGetChildrenString from '../../helpers/recursivelyGetChildrenString'
+import { Color } from '../../theme/colors'
+import Box from '../Box'
+import Tooltip, { TooltipProps } from '../Tooltip'
+
+type StyleProps = {
+  theme?: Theme
+  maxLines?: string | number
```

Comment:
`maxLine` is defined as a number everywhere else in this file

```suggestion
  maxLines?: number
```